javaslang/src/test/java/javaslang/collection/euler/Euler17Test.java

```diff
@@ -99,7 +99,7 @@ default int letterCount(Seq<Integer> range) {
             1_000, "thousand",
             1_000_000, "million",
             1_000_000_000, "billion"
-    );
+    ).grouped(2).toSortedMap(pair -> Tuple.of((Integer) pair.get(0), (String) pair.get(1)));
```

Comment:
Looks good to me. As Ruslan says, it works - we should spend our energy on the issues.